src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordCopyCtor.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedRecordCopyCtor : SynthesizedInstanceConstructor
+    {
+        public SynthesizedRecordCopyCtor(
+            SourceMemberContainerTypeSymbol containingType,
+            DiagnosticBag diagnostics)
+            : base(containingType)
+        {
+            Debug.Assert(!containingType.IsStructType(), "only reference types should define copy constructors");
+            Parameters = ImmutableArray.Create(SynthesizedParameterSymbol.Create(
+                this,
+                TypeWithAnnotations.Create(
+                    isNullableEnabled: true,
```

Comment:
>isNullableEnabled: true, [](start = 20, length = 24)

nit: did we need this?

Reply:
I'm not sure. Is there any reason why it wouldn't be considered nullable enabled? We know what the correct annotation is.